### DISCUSSION TITLE
There's a Nexon font for the squad type (Striker, Special)

### DISCUSSION THREAD

**Discussion by OkazakiNaoki** [lonqie/SchaleDB#52] [Ideas and Suggestions]
https://levelup.nexon.com/index.html
It's downloadable at carousel 7/8. (Nexon football gothic B)
Maybe you can put them on.

**Comment by lonqie** [comment]
Thanks for letting me know about this, I will consider using this font in the future

**Reply by OkazakiNaoki** [reply]
And there's a woff version here.
https://wess.tistory.com/196
Otherwise windows version is ttf and mac version is otf at official site.
